packages/lib/gpc/src/gpcCompile.ts

```diff
@@ -387,6 +394,71 @@ function compileProofObject(objInfo: CompilerObjInfo<POD>): ObjectModuleInputs {
   };
 }
 
+function compileProofBoundsChecks<
+  ObjInput extends POD | GPCRevealedObjectClaims
+>(
+  boundsCheckConfig: GPCProofBoundsCheckConfig,
+  entryMap: Map<PODEntryIdentifier, CompilerEntryInfo<ObjInput>>,
+  paramBoundsChecks: number
+): {
+  boundsCheckEntryIndices: CircuitSignal[];
+  boundsCheckMinValues: CircuitSignal[];
+  boundsCheckMaxValues: CircuitSignal[];
+} {
+  // Arrange POD entry identifiers alphabetically
+  const boundsCheckIdOrder = Object.keys(
+    boundsCheckConfig
+  ).sort() as PODEntryIdentifier[];
+
+  // Compile entry indices
+  const unpaddedBoundsCheckEntryIndices = boundsCheckIdOrder.map((entryId) => {
+    const idx = entryMap.get(entryId)?.entryIndex;
+
+    if (idx === undefined) {
+      throw new Error(`Missing input for identifier ${entryId}.`);
```

Comment:
ReferenceError?
Mind you, I think given how these inputs are derived, this shouldn't happen due to user input, but would indicate an internal bug.